<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TsHelper</class>
 <widget class="QMainWindow" name="TsHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OcrSlicer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SegmenterView" name="sceneview">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::SmartViewportUpdate</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="TranscriptEdit" name="lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshText">
        <property name="toolTip">
         <string>Reload the text from file</string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Bookstore"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Transcript"/>
    <addaction name="actionSave_Transcript"/>
    <addaction name="actionSave_Transcript_as"/>
    <addaction name="separator"/>
    <addaction name="actionWrite"/>
    <addaction name="actionExport_CSeg_PNGs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWiki"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPrev"/>
    <addaction name="actionNext"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionFit_Width"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAuto_Guess"/>
    <addaction name="actionAuto_Write"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionGuess"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawMode"/>
   <addaction name="actionMoveMode"/>
   <addaction name="actionDeleteMode"/>
   <addaction name="separator"/>
   <addaction name="actionPrev"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionGuess"/>
   <addaction name="actionPreview"/>
   <addaction name="actionWrite"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPrev">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionDrawMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionMoveMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionDeleteMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionGuess">
   <property name="text">
    <string>WS Guess</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="text">
    <string>Export GT</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="actionAuto_Guess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Guess</string>
   </property>
  </action>
  <action name="actionAuto_Write">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Export GT</string>
   </property>
  </action>
  <action name="actionOpen_Transcript">
   <property name="text">
    <string>Open CSeg XML...</string>
   </property>
  </action>
  <action name="actionSave_Transcript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save CSeg XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Transcript_as">
   <property name="text">
    <string>Save CSeg XML As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen_Bookstore">
   <property name="text">
    <string>Open Bookstore...</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Wiki page on Google Code...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionExport_CSeg_PNGs">
   <property name="text">
    <string>Export All GTs...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SegmenterView</class>
   <extends>QGraphicsView</extends>
   <header>segmenterview.h</header>
  </customwidget>
  <customwidget>
   <class>TranscriptEdit</class>
   <extends>QLineEdit</extends>
   <header>transcriptedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>TsHelper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
